src/Chapter2.hs

```diff
@@ -620,7 +652,10 @@ Write a function that takes elements of a list only on even positions.
 >>> takeEven [2, 1, 3, 5, 4]
 [2,3,4]
 -}
-takeEven = error "takeEven: Not implemented!"
+takeEven :: [a] -> [a]
+takeEven (x : _ : xs) = x : takeEven xs
+takeEven (x : _) = [x]
+takeEven [] = []
```

Comment:
Very nice and idiomatic solutions! You've mastered pattern-matching on lists :train: :train: :train: